atsign-foundation/noports
# fix: 902 noports desktop enhancements v4

**- What I did**

- Updated the app so the font size and some widgets size will increase up to a maximum point based the device size.
- Updates app assets with the new no ports logo
- Update app launcher icon on macOS

closes #902

**- How I did it**

**- How to verify it**
Run the app and increase the windows size, you should observe the app font size increasing up to a certain point.

**- Description for the changelog**

App Scales based on device Size
App uses new no ports logos
